### PR TITLE
Footer Component

### PR DESCRIPTION
**Pull request by yosefanajjar**
**Issue**: #24 

## What did I do?
Created the footer component as mentioned in the issue:

- [x] Create the footer component
- [x] Style the footer component
- [x] Add media query for smaller screens
- [x] Make a storybook of the component

## How to test?
You can test the footer component by opening the **first page** because it's already in the layout and there is also a **storybook** that I made for it

### CODE REVIEW

File: src/components/layout/Footer/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      <H4>
+        <span>3.0</span>Our Story + Team
+      </H4>
+      <H4>
+        <span>4.0</span>Get in touch
+      </H4>
+    </SitemapSection>
+    <ContactSection>
+      <CapsHeadingSmall>contact</CapsHeadingSmall>
+      <EmailLink>hello@yallacooperative.com</EmailLink>
```
Comment (Amoodaa) [inline src/components/layout/Footer/index.js]: you could use an anchor ? with  `mailto:`

Comment (yosefanajjar) [reply src/components/layout/Footer/index.js]: I know and I wanted to do it but it wasn't like mentioned or anything

**Comment by yosefanajjar** [comment]
@jema28 can you check the spacing in the footer and see if it's like we want?

File: src/components/layout/Footer/index.js [hunk 2]
```diff
@@ -32,7 +32,12 @@ const Footer = () => (
     </SitemapSection>
     <ContactSection>
       <CapsHeadingSmall>contact</CapsHeadingSmall>
-      <EmailLink>hello@yallacooperative.com</EmailLink>
+      <EmailLink
+        href="mailto:hello@yallacooperative.com?Subject=Hello%20again"
+        target="_top"
```
Comment (Amoodaa) [inline src/components/layout/Footer/index.js]: use the `rel`? security i think, i dont think it matters personally

**Comment by fadeomar** [comment]
it's ready to merge

File: src/components/layout/Footer/index.js (hunk 2 above)
Comment (yosefanajjar) [reply src/components/layout/Footer/index.js]: yea I think here its fine to not use it

**Pull request merged**